x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/translations.ts

```diff
@@ -13,7 +13,7 @@ export const OVERVIEW = i18n.translate('xpack.securitySolution.endpointDetails.o
 
 export const ACTIVITY_LOG = {
   tabTitle: i18n.translate('xpack.securitySolution.endpointDetails.activityLog', {
```

Comment:
Please change the translation key as well for consistency. We forgot to do this earlier I think. Maybe call it `...endpointDetails.responseActionsHistory`

Reply:
updated